classes/plugins/Plugin.php

```diff
@@ -543,7 +542,7 @@ public function _overridePluginTemplates($hookName, $args)
      *
      * @return string|null
      */
-    private function _findOverriddenTemplate($path)
+    public function _findOverriddenTemplate($path)
```

Comment:
No problems with moving this to `public`, but please also remove the `_` prefix from the function name.